client/src/App.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import './App.css';
+import { NavLink, Route } from 'react-router-dom';
+import Users from './users/Users.js';
```

Comment:
My computer threw an error because `users` should be capitalized in `'./users/Users.js'` to make it `'./Users/Users.js'`. Double check this in your projects so that your app will run on all machines.